CMakeLists.txt

```diff
@@ -19,6 +19,10 @@ else(AKTUALIZR_VERSION)
     add_definitions(-DAKTUALIZR_VERSION="dev")
 endif(AKTUALIZR_VERSION)
 
+if(PROVISION_FILE)
+    add_definitions(-DPROVISION_FILE="${PROVISION_FILE}")
+endif(PROVISION_FILE)
+
```

Comment:
I'm not keen on compiling test code into the build output.  Could we instead pass this in to the test cases via the command line?